src/igvfd/audit/file.py

```diff
@@ -155,3 +155,42 @@ def audit_file_no_file_format_specifications(value, system):
             f'has no `file_format_specifications`.'
         )
         yield AuditFailure(audit_message.get('audit_category', ''), f'{detail} {audit_message.get("audit_description", "")}', level=audit_message.get('audit_level', ''))
+
+
+@audit_checker('SequenceFile', frame='object')
+def audit_multiple_seqspec_per_seqfile(value, system):
+    '''
+    [
+        {
+            "audit_description": "A sequence file is expected to have only one released sequence specification file.",
+            "audit_category": "multiple seqspecs per sequence file",
+            "audit_level": "ERROR"
+        },
+        {
+            "audit_description": "A sequence file is expected to have only one in progress sequence specification file.",
+            "audit_category": "multiple seqspecs per sequence file",
+            "audit_level": "INTERNAL_ACTION"
+        }
+    ]
+    '''
+    audit_msg_multi_released_seqspec = get_audit_message(audit_function=audit_multiple_seqspec_per_seqfile, index=0)
+    audit_msg_multi_inprogress_seqspec = get_audit_message(audit_function=audit_multiple_seqspec_per_seqfile, index=1)
+    seqspec_files = value.get('seqspecs', '')
+    if len(seqspec_files) > 1:
+        all_seqspec_file_status = []
+        for seqspec_file in seqspec_files:
+            seqspec_file_obj = system.get('request').embed(seqspec_file, '@@object?skip_calculated=true')
+            seqspec_file_status = seqspec_file_obj.get('status', '')
+            all_seqspec_file_status.append(seqspec_file_status)
+        if all_seqspec_file_status.count('released') > 1:
+            detail = (
+                f'Sequence File {audit_link(path_to_text(value["@id"]), value["@id"])} '
+                f'has multiple released sequence specification files.'
+            )
+            yield AuditFailure(audit_msg_multi_released_seqspec.get('audit_category', ''), f'{detail} {audit_msg_multi_released_seqspec.get("audit_description", "")}', level=audit_msg_multi_released_seqspec.get('audit_level', ''))
+        elif all_seqspec_file_status.count('in progress') > 1:
```

Comment:
I think this `elif` can be changed to `if`, since it is hypothetically possible for both audits to occur. (Imagine one file with 2 released seqspecs and 2 in progress seqspecs)